docs/Using-Fleet/Permissions.md

```diff
@@ -6,39 +6,39 @@ Users with the Admin role receive all permissions.
-| View all policies                                    | ✅       | ✅         | ✅    |
-| Filter hosts using policies                          | ✅       | ✅         | ✅    |
-| Create, edit, and delete policies for all hosts      |          | ✅         | ✅    |
-| Create, edit, and delete policies for all hosts assigned to team\*     |          | ✅         | ✅    |
-| Manage policy automations      |          |           | ✅    |
-| Create, edit, view, and delete users                       |          |            | ✅    |
-| Add and remove team members\*                        |          |            | ✅    |
-| Create, edit, and delete teams\*                     |          |            | ✅    |
-| Create, edit, and delete enroll secrets              |          | ✅         | ✅    |
-| Create, edit, and delete enroll secrets for teams\*  |          | ✅         | ✅    |
-| Edit organization settings                           |          |            | ✅    |
-| Edit agent options                                   |          |            | ✅    |
-| Edit agent options for hosts assigned to teams\*     |          |            | ✅    |
+| **Action**                                                                         | Observer | Maintainer | Admin |
+| ----------------------------------------------------                               | -------- | ---------- | ----- |
+| View all activity                                                                  | ✅       | ✅         | ✅    |
+| View all hosts                                                                     | ✅       | ✅         | ✅    |
+| Filter hosts using labels                                                          | ✅       | ✅         | ✅    |
+| Target hosts using labels                                                          | ✅       | ✅         | ✅    |
+| Add and delete hosts                                                               |          | ✅         | ✅    |
+| Transfer hosts between teams\*                                                     |          | ✅         | ✅    |
+| Create, edit, and delete labels                                                    |          | ✅         | ✅    |
+| View all software                                                                  | ✅       | ✅         | ✅    |
+| Filter software by vulnerabilities                                                 | ✅       | ✅         | ✅    |
+| Filter hosts by software                                                           | ✅       | ✅         | ✅    |
+| Filter software by team\*                                                          | ✅       | ✅         | ✅    |
+| Manage vulnerability automations                                                   |          |            | ✅    |
+| Run only designated, _observer can run_ ,queries as live queries against all hosts | ✅       | ✅         | ✅    |
```

Comment:
```suggestion
| Run only designated, **observer can run** ,queries as live queries against all hosts | ✅       | ✅         | ✅    |
```
Still updating the style guide, but please avoid using italics for emphasis. Use bold instead.